custom_components/favicon/__init__.py

```
@@ -80,11 +77,13 @@
-                    "sizes": icon.group(1),
-                    "type": "image/png",
-                    })
+                manifest.append(
+                    {
+                        "src": os.path.join(path, fn),
+                        "sizes": icon.group(1),
+                        "type": "image/png",
+                    }
+                )
                 _LOGGER.info("Found icon: %s", os.path.join(path, fn))
     except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)